### PR TITLE
New MTurk timeout error message; fix for giving qualifications

### PR DESCRIPTION
**Pull request by klshuster**
Updated the timeout message so that in tasks with no inter-turker communication, the turker is not confused by the timeout message.

### CODE REVIEW

File: parlai/mturk/core/mturk_manager.py [hunk 1]
```diff
@@ -1310,8 +1310,7 @@ def soft_block_worker(self, worker_id):
         """Soft block a worker by giving the worker the block qualification"""
         qual_name = self.opt['block_qualification']
         assert qual_name != '', ('No block qualification has been specified')
-        self.give_worker_qualification(worker_id, qual_name,
-                                       is_sandbox=self.is_sandbox)
+        self.give_worker_qualification(worker_id, qual_name)
```
Comment (JackUrb) [inline parlai/mturk/core/mturk_manager.py]: Why did this change?

**Comment by JackUrb** [comment]
This is looking good to me! Other than that one comment

File: parlai/mturk/core/mturk_manager.py (hunk 1 above)
Comment (klshuster) [reply parlai/mturk/core/mturk_manager.py]: ``is_sandbox`` is not a keyword arg for ``self.give_worker_qualification`` (though it is an argument for the ``mturk_client``; that's handled in ``self.give_worker_qualification``).

Comment (JackUrb) [reply parlai/mturk/core/mturk_manager.py]: Oh oops I don't think I've ever directly used manager.give_worker_qualification, hence the bug. Thanks for the fix!

**Pull request merged**